apps/air-discount-scheme/backend/src/app/modules/flight/flight.model.ts

```diff
@@ -77,7 +77,7 @@ export class FlightLeg extends Model<FlightLeg> implements TFlightLeg {
 
   // eslint-disable-next-line
   @BelongsTo(() => Flight)
-  flight
+  flight: any
```

Comment:
The app crashes it has the Flight type, there is a circular dependency going on here because Flight hasMany FlightLegs and FlightLeg has one Flight.
@darrikonn Do we need Flight here? Are you referencing Flight from a FlightLeg in the admin view?

Reply:
Yup, need it both in `admin` and `min-rettindi`.